src/test/java/se/sundsvall/document/api/model/DocumentDataUpdateRequestTest.java

```diff
@@ -33,16 +33,18 @@ void testBuilderMethods() {
 
 		final var bean = DocumentDataUpdateRequest.create()
 			.withConfidentiality(confidentiality)
-			.withCreatedBy(createdBy);
+			.withCreatedBy(createdBy)
+			.withArchive(false);
 
 		assertThat(bean).isNotNull().hasNoNullFieldsOrProperties();
 		assertThat(bean.getConfidentiality()).isEqualTo(confidentiality);
 		assertThat(bean.getCreatedBy()).isEqualTo(createdBy);
+		assertThat(bean.getArchive()).isFalse();
 	}
 
 	@Test
 	void testNoDirtOnCreatedBean() {
-		assertThat(DocumentDataUpdateRequest.create()).hasAllNullFieldsOrProperties();
-		assertThat(new DocumentDataUpdateRequest()).hasAllNullFieldsOrProperties();
+		assertThat(DocumentDataUpdateRequest.create()).hasAllNullFieldsOrPropertiesExcept("archive");
```

Comment:
As "archive" is of type Boolean it should be possible to remove the except-part here, right?